### ISSUE TITLE
Crash when clicking into configurations list

### ISSUE THREAD

**Issue by OliverUrbann** [midwan/amiberry#750]
On my Raspberry Pi 400 with Raspberry Pi OS (32 bit) everything up to date (today) I get the following crash when I click into the list of configurations:

```
INFO: --- New exception ---
INFO: JIT not in use.
INFO: Segmentation Fault
INFO: info.si_signo = 11
INFO: info.si_errno = 0
INFO: info.si_code = 1
INFO: info.si_addr = 00000010
INFO: r0  = 0x60000981
INFO: r1  = 0x00000001
INFO: r2  = 0x00000000
INFO: r3  = 0x00000320
INFO: r4  = 0x03b456b8
INFO: r5  = 0x03b240f0
INFO: r6  = 0xbe93d820
INFO: r7  = 0x03b24148
INFO: r8  = 0x03b24128
INFO: r9  = 0x00665d30
INFO: r10 = 0x03b4a3b0
INFO: FP  = 0xbe93e354
INFO: IP  = 0x00000258
INFO: SP  = 0xbe93d7c0
INFO: LR  = 0x0020c1e0
INFO: PC  = 0x0020c1ec
INFO: CPSR = 0x68000010
INFO: Fault Address = 0x00000010
INFO: Trap no = 0x0000000e
INFO: Err Code = 0x00000207
INFO: Old Mask = 0x00000000
INFO: LR - 0x0020C1E0: <(null)> (./amiberry)
INFO: Stack trace:
INFO: 0x00208d20 <(null) + 0x00208d20> (./amiberry)
INFO: 0xb66f0130 <__default_rt_sa_restorer + 0x00000000> (/lib/arm-linux-gnueabihf/libc.so.6)
INFO: 0x0020c1ec <(null) + 0x0020c1ec> (./amiberry)
INFO: 0x001ff588 <(null) + 0x001ff588> (./amiberry)
INFO: 0x0016aefc <(null) + 0x0016aefc> (./amiberry)
INFO: 0x0016f94c <(null) + 0x0016f94c> (./amiberry)
INFO: 0x00165be0 <(null) + 0x00165be0> (./amiberry)
INFO: 0x0001876c <main + 0x000001a8> (./amiberry)
INFO: 0xb66da718 <__libc_start_main + 0x0000010c> (/lib/arm-linux-gnueabihf/libc.so.6)
INFO: 01: 0x03b4a3b0 <(null) + 0x03b4a3b0> (./amiberry)
INFO: Stack trace (non-dedicated):
INFO: ./amiberry() [0x208e38]
INFO: /lib/arm-linux-gnueabihf/libc.so.6(__default_rt_sa_restorer+0) [0xb66f0130]
INFO: ./amiberry() [0x20c1ec]
INFO: ./amiberry() [0x1ff588]
INFO: ./amiberry() [0x16aefc]
INFO: ./amiberry() [0x16f94c]
INFO: ./amiberry() [0x165be0]
INFO: ./amiberry(main+0x1a8) [0x1876c]
INFO: /lib/arm-linux-gnueabihf/libc.so.6(__libc_start_main+0x10c) [0xb66da718]
INFO: End of stack trace.
INFO: --- end exception ---
```

I checked out the recent version, compiled it with `make -j4 PLATFORM=rpi4` and start it with `./amiberry`.

**Comment by midwan** [comment]
I'm assuming you got the version from `master`, which is currently v3.3.
Was it an empty Config list? If so, this is a known bug that was fixed in the `dev` branch, for the upcoming v3.4.

**Comment by OliverUrbann** [comment]
Thx! Didn't know that development is done on a branch, the branch doesn't crash.
